advanced_functionality/autogluon-tabular/AutoGluon_Tabular_SageMaker.ipynb

```diff
@@ -76,8 +76,8 @@
     "    )\n",
```

Comment:
<p>Build fails in this cell. Error is:</p><pre class="ql-syntax" spellcheck="false">s3://sagemaker-us-west-2-521695447989/autogluon-sagemaker-training-2021-03-11-04-17-28-887/output/model.tar.gz"
</pre><p> </p><p>my guess is the <code>model_uri</code>  you defined as <code>os.path.join(estimator.output_path, estimator._current_job_name, "output", "model.tar.gz")</code>  is not where the model is. Try to use <code>estimator.model_data</code>  directly. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2053/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
Fixed. 
model_uri = estimator.model_data. See commit : bcf58a6